habitat-lab/habitat/config/benchmark/rearrange/cat_gaze.yaml

```diff
@@ -44,10 +44,12 @@ habitat:
              height: 256
              width: 256
              hfov: 42
+             position: [0,1.31,0]
            head_panoptic_sensor:
              height: 256
              width: 256
              hfov: 42
+             position: [0,1.31,0]
```

Comment:
I assume you used something like this before to set the height. I never tested this before

Reply:
Yes I have used this in Hab1.0 task. But now thinking more, I dont know if the URDF plays any role in finding out the height of the camera for Hab2.0 tasks.